src/Controller/AutologoutController.php

```diff
@@ -21,7 +21,7 @@ class AutologoutController extends ControllerBase {
    * AJAX callback that performs the actual logout and redirects the user.
```

Comment:
Let's use a variable called `$autologoutManager` at class level, and initialise it in the constructor of this class as

`$this->autologoutManager = \Drupal::service('autologout.manager');`

Then this variable could be used throughout the class.